www/css/site.scss

```diff
@@ -195,7 +196,7 @@ h4 {
         display: none !important;
     }
     .hamburger {
-        display: inline-block;
+        // display: inline-block;
```

Comment:
Instead of commenting, maybe delete?

Reply:
Oh that definitely was my intention. Not sure why I commented. 👍🏽